src/react-marketplace/components/hooks/useTile.tsx

```diff
@@ -30,6 +30,17 @@ const useTile = ({
 
   const entity = useMemo(() => feed.find((e) => e.id === feedId), [feed, feedId]);
 
+  const { subtitle, description } = useMemo(() => {
+    const tags = entity?.tags.catalog;
+    const desc = entity?.description;
+
+    const parsedTags = tags?.replaceAll(',', ', ');
+    const descriptionParagraph = desc?.split('[')?.[0];
+    console.log(descriptionParagraph);
```

Comment:
@MikeFuster Let's remove this console log

Reply:
Removed